### DISCUSSION TITLE
为研究生/博士生论文模板添加**南京大学学位论文使用授权声明**

### DISCUSSION THREAD

**Discussion by dreistein543** [nju-lug/NJUThesis#95] [Ideas]
建议在**南京大学学位论文原创性声明**同一页展示**授权声明**，格式与原创性声明相同，只是多了“导师签名”及“日期”
---
授权声明全文如下：
本人完全了解南京大学有关保留、使用学位论文的规定，学校有权保留学位论文并向国家主管部门或其指定机构送交论文的电子版和纸质版。有权将学位论文用于非赢利目的的少量复制并允许进入学校及院系图书馆被查询。有权将学位论文的内容编入有关数据库进行检索。有权将学位论文的标题和摘要汇编出版。保密的学位论文在解密后适用本规定。
作者签名：日期
导师签名：日期

**Comment by atxy-blip** [comment]
这个有来源吗？我手头找到的是下面这个声明文件：

![image](https://user-images.githubusercontent.com/63113546/157812638-c9614bb6-f170-4641-b657-1bd7b7b87fb5.png)

**Reply by dreistein543** [reply]
其实只是参考了学长学姐的毕业论文格式，商学院的全文如上所写，这里有个医学院的正式说明[南京大学医学院博士学位论文撰写与印制要求 (供参考)](https://med.nju.edu.cn/a0/6d/c10802a434285/page.htm)

**Reply by atxy-blip** [reply]
唉，好。周末就先加上商院这个简单点的吧。真希望研究生院能出来统一度量衡。

**Reply by atxy-blip** [reply]
已增加，见 <https://github.com/nju-lug/NJUThesis/issues/116>。

**Reply by dreistein543** [reply]
如我在 #116 中回复的

> 目前该页还有一个问题，会受到之前章节的影响添加页眉，以及被添加上了不需要的页码，例如在加上致谢之后，该页面会被添加上致谢的页眉，以及页码如下图：
> <img width="764" alt="截屏2022-04-14 20 20 57" src="https://user-images.githubusercontent.com/30745407/163389357-cf85c997-9364-47c5-8d45-8710cf8c35b0.png">
> <img width="875" alt="截屏2022-04-14 20 46 54" src="https://user-images.githubusercontent.com/30745407/163393414-92cb8cf8-2969-4336-849d-52e76f9312b9.png">
